app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<div>
+  <div>
+    <h5>Public Recipes</h5>
+  </div>
+  <div>
+    <% if @public_recipes.count == 0 %>
+      <h5>Sorry we don't have any public recipe</h5>
+    <% else %>
+      <% @public_recipes.each do |recipe| %>
+        <div>
+          <div>
+            <div>
+              <%= link_to recipe_path(recipe[:id]), method: :get, class: "nav-link" do %>
+                <h5>
+                  <%= recipe[:recipe_name] %>
+                </h5>
+              <% end %>
+            </div>
+            <div>
+              <h5>By <%= recipe[:author].name %></h5>
+            </div>
+          </div>
+          <div>
+            <h5>Total food items : <%= recipe[:food_items] %> </h5>
+            <h5>Total price : $ <%= recipe[:total_price] %></h5>
+          </div>
+        </div>
+      <% end %>
+    <% end %>
+  </div>
```

Comment:
- [ ] Hi team, I got this console error when I click on the `public Recipes` link. Here is the screenshot. Please fix it.


![Screenshot from 2022-07-29 23-50-29](https://user-images.githubusercontent.com/63087491/181855883-be24d693-d605-4e6b-9fc8-c498c9f3bea8.png)